src/EUTelProcessorAnalysisPALPIDEfs.cc

```diff
@@ -70,16 +70,17 @@ EUTelProcessorAnalysisPALPIDEfs::EUTelProcessorAnalysisPALPIDEfs()
   _minTimeStamp(0),
   _showFake(true),
   _realAssociation(false),
-  nTracks(4),
-  nTracksFake(4),
-  nTracksPAlpide(4),
-  nTracksPAlpideFake(4),
-  nTracksAssociation(4),
-  nTracksPAlpideAssociation(4),
-  nFakeWithTrack(4,0),
-  nFakeWithoutTrack(4,0),
-  nFake(4,0),
-  nFakeWithTrackCorrected(4,0),
+  nTracks(8),
+  nTracksPAlpide(8),
+  nTracksFake(8),
+  nTracksPAlpide(8),
```

Comment:
@mkofarag You mean nFake and nTracksFake is same value?
nTracksFake series are the one that is new feature introduced by you and your colleague,
and nFake was the one from original version.